apps/projects/app/components/Content/IssueDetail/components/Detail.js

```diff
@@ -242,16 +256,18 @@ const activities = (requestsData, workSubmissions, onReviewApplication, onReview
             'rejected ' + data.user.login + '\'s work',
           eventAction: data.review.feedback.length === 0 ?
             <Badge
-              background={data.review.accepted ? '#e7f8ec' : '#f3c1c3' }
-              foreground={data.review.accepted ? theme.positive : theme.negative}
+              foreground={theme.textSecondary}
```

Comment:
Which branch do we favor for `background` and `foreground` of `Badge` component here? Wondering if this got overwritten or if you meant to make this change

Reply:
It should be grey for now, yes.